ruby/.rubocop.yml

```diff
@@ -53,6 +53,8 @@ Metrics/ClassLength:
 
 Metrics/LineLength:
   Max: 150
+  Exclude:
+    - '**/*_test.rb'
```

Comment:
if we're going to take this approach, then can we also exclude `'**/*_spec.rb` files? some of our gems run spec files (i.e. autobahn)

Reply:
actually..autobahn is all spec files...probably shouldn't do that then